src/compiler/crystal/macros.cr

```diff
@@ -883,6 +887,9 @@ module Crystal::Macros
   #
   # Its macro methods are nearly the same as `ArrayLiteral`.
   class TupleLiteral < ASTNode
+    # Similar to `Array#-`.
+    def -(other : ArrayLiteral) : ArrayLiteral
```

Comment:
Is it worth breaking convention for this one method? Given in the past we just add the methods to `ArrayLiteral` and `TupleLiteral` links to it.

Reply:
Yeah, I think it's worth to overthrow that convention and *actually* document `TupleLiteral`. It doesn't need to happen in this PR, though.